bindings/python/readme.md

```diff
@@ -1,63 +1,20 @@
 # 3D Time of Flight : Python bindings
 
 ### Overview
-The SDK provides bindings for Python using [pybind11](https://github.com/pybind/pybind11). To use the bindings for a Python project, just import the library:
-```python
-import aditofpython as tof
-```
-
-Initialize a system
-```python
-system = tof.System()
-status = system.initialize()
-```
-
-Getting a camera from the system, configuring the mode and the frame type and initializing it
-```python
-cameras = []
-status = system.getCameraList(cameras)
-print("system.getCameraList()", status)
-
-camera1 = cameras[0]
+The SDK provides bindings for Python using [pybind11](https://github.com/pybind/pybind11). 
 
-modes = []
-status = camera1.getAvailableModes(modes)
-print("system.getAvailableModes()", status)
-print(modes)
-
-types = []
-status = camera1.getAvailableFrameTypes(types)
-print("system.getAvailableFrameTypes()", status)
-print(types)
-
-status = camera1.initialize()
-print("camera1.initialize()", status)
-
-camDetails = tof.CameraDetails()
-status = camera1.getDetails(camDetails)
-print("system.getDetails()", status)
-print("camera1 details:", "id:", camDetails.cameraId, "connection:", camDetails.connection)
-
-status = camera1.setFrameType(types[0])
-print("camera1.setFrameType()", status)
-
-status = camera1.setMode(modes[0])
-print("camera1.setMode()", status)
-```
-
-Getting a frame from the camera
+To use the bindings for a Python project, just import the library:
 ```python
-frame = tof.Frame()
-status = camera1.requestFrame(frame)
-print("camera1.requestFrame()", status)
+import aditofpython as tof
 ```
 
-
 #### Directory Structure
 
 | Directory/File | Description |
 | --------- | ----------- |
 | cmake | Contains the cmake files to find pybind11 |
 | pybind11 | Pybind11 header only library |
 | CMakeLists.txt | Rules to build the Python bindings |
-| first_frame.py | Python example to capture a frame |
+| first_frame | Python example to capture a frame |
+| dnn | Simple object detection example in Python |
+| device_example | Simple Python example of using Device Manager |
```

Comment:
Not sure where 'Device Manager' is coming from? It should say 'DeviceInterface' or 'device from the low-level API'

Reply:
implemented with commit: e3a28ef